includes/templates/contact.php

```diff
@@ -59,7 +59,7 @@
 		'form_title'                         => $form_title,
 		'form_desc'                          => '',
 		'successful_form_submission_message' => get_option( 'everest_forms_successful_form_submission_message', __( 'Thanks for contacting us! We will be in touch with you shortly.', 'everest-forms' ) ),
-		'redirect_to'                        => '0',
+		'redirect_to'                        => 'same',
```

Comment:
@milan1750 Nice catch as we have changed the `redirect_to` logic previously but didn't changed it in the default contact form template 👍